packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -29,6 +30,9 @@ const SECONDS_PER_DAY = 24n * SECONDS_PER_HOUR;
 const BASIS_POINTS = 10_000n;
 const MILLI = 1_000_000n;
 
+const hardenDeeplyFullfilled = async expression =>
+  deeplyFulfilled(harden(expression));
```

Comment:
This barely saves any typing and the other files use the two functions. My best guess why this is defined here is so the contets of `ammGovernorTerms` don't need to be reindented. But if so then packages/inter-protocol/src/vaultFactory/params.js wouldn't be re-indented.

Please consider using this helper there or removing it entirely

Reply:
I agree it barely saves typing, but it does save a layer of indentation, and for `terms` we should always use both together. I've moved it to Zoe/contractSupport, and renamed it to `deeplyFulfillTerms`.